### PR TITLE
Update README.MD on coffee-node

### PR DESCRIPTION
**Pull request by osmarpetry**
# Pull Request

## What I did
Create a new README.MD to coffee-node formula

## How I did it
Run it on the local environment, then I edited the README.MD

## How to verify it
Run the same commands

## What kind of change does this PR make

- [ ] Major
- [ ] Minor
- [X] Patch

## Description for the changelog

New fully updated README.MD to coffee-node formula

### CODE REVIEW

**Comment by osmarpetry** [comment]
It solves #5

**Comment by GuillaumeFalourd** [comment]
Thank you fo contributing @osmarpetry 👍🏼 
You'll just need to sign-off the DCO before we could merge your PR.

File: demo/coffee-node/README.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
 # coffee-node
 
-## command
+## Premisses
+
+- [Node.js installed](https://nodejs.org/)
```
Comment (GuillaumeFalourd) [inline demo/coffee-node/README.md:5]: I would had that Node.js is necessary if the execution is local. Through Docker this is not necessary.

**Review by GuillaumeFalourd — APPROVED** [review]

**Comment by GuillaumeFalourd** [comment]
Thank you for contributing @osmarpetry 👍 

We want to show our appreciation by sending you some swag. Could you please fill out [this form](https://zup1.typeform.com/to/zrOF5cpp) so we can send it to you? 🚀

**Pull request merged**